src/styles/card.scss

```diff
@@ -0,0 +1,109 @@
+html{
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center; /* Центрування по горизонталі */
+  background-color: darkgray; /* Колір фону */
+  height: 100vh;
+  align-items: center;
+}
+
+.product-card {
+  background-color: $main-page-color;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+
+  &__image {
+    background-repeat: no-repeat;
+    background-position: center top;
+    margin-top: 32px;
+    background-image: url("../images/imac.jpg");
+    height: 134px;
+    width: 100%;
+  }
+
+
+  &__content {
+    margin: 0 16px;
+  }
+}
+
+  .main__text {
```

Comment:
The class name `.main__text` does not follow the BEM naming convention. It should be prefixed with the block name, such as `.product-card__content__main-text`.